libraries/AP_Logger/LogStructure.h

```diff
@@ -1385,6 +1400,19 @@ struct PACKED log_PSCZ {
 // @Field: CAn: Angle to closest object
 // @Field: CDis: Distance to closest object
 
+// @LoggerMessage: PRXR
+// @Description: Proximity Raw sensor data
```

Comment:
```suggestion
// @Description: Raw proximity sensor data
```

Reply:
Again, I'd actually leave Proximity at the front..